contracts/chain-adapters/ZkSync_Adapter.sol

```diff
@@ -53,6 +53,63 @@ interface ZkBridgeLike {
     ) external payable returns (bytes32 txHash);
 }
 
+// Note: this contract just forwards the calls from the HubPool to ZkSync to avoid limits.
+// A modified ZKSync_Adapter should be deployed with this address swapped in for all zkSync addresses.
+contract LimitBypassProxy is ZkSyncInterface, ZkBridgeLike {
+    using SafeERC20 for IERC20;
+    ZkSyncInterface public constant zkSync = ZkSyncInterface(0x32400084C286CF3E17e7B677ea9583e60a000324);
```

Comment:
`Solhint` is indicating that this should be in uppercase snake case `ZKSYNC`. Should we modify the solhint?

Reply:
@james-a-morris @pxrl Since we generally don't snake case contract interfaces, I removed the solhint warning for now to appease the linter.